layers/meta-balena-genericx86/recipes-kernel/linux/linux-yocto_%.bbappend

```diff
@@ -419,6 +422,19 @@ BALENA_CONFIGS[igc] = " \
     CONFIG_IGC=m \
 "
 
+BALENA_CONFIGS_append_genericx86-64-ext = " secureboot"
+BALENA_CONFIGS[secureboot] = " \
+    CONFIG_LOCK_DOWN_IN_EFI_SECURE_BOOT=y \
+    CONFIG_MODULE_SIG=y \
+    CONFIG_MODULE_SIG_ALL=y \
+    CONFIG_MODULE_SIG_FORCE=y \
+    CONFIG_MODULE_SIG_SHA512=y \
+    CONFIG_SECURITY_LOCKDOWN_LSM=y \
+    CONFIG_SECURITY_LOCKDOWN_LSM_EARLY=y \
+    CONFIG_SYSTEM_TRUSTED_KEYS="certs/balenaos.crt" \
+"
+
+
```

Comment:
Some of these are architecture independent and should move to meta-balena. Probably all except `CONFIG_LOCK_DOWN_IN_EFI_SECURE_BOOT`